build/Build.proj

```diff
@@ -34,13 +34,16 @@
 	<Target Name="SetVersionNumber" DependsOnTargets="CleanUp" Condition="'$(BUILD_RELEASE)'!=''">		
 		<!--This updates the copyright year-->
 		<FileUpdate
-			Files="..\src\Umbraco.Deploy.Contrib.Connectors\Properties\AssemblyInfo.cs"
+			Files="..\src\Umbraco.Deploy.Contrib.Connectors\Properties\AssemblyInfo.cs;..\src\Umbraco.Deploy.Contrib.Export\Properties\AssemblyInfo.cs"
 			Regex="AssemblyCopyright\(&quot;(.+)?&quot;\)"
 			ReplacementText="AssemblyCopyright(&quot;Copyright © Umbraco $([System.DateTime]::Now.ToString(`yyyy`))&quot;)"/>
 		<!--This updates the assembly versions-->
 		<AssemblyInfo CodeLanguage="CS"
 			OutputFile="..\src\Umbraco.Deploy.Contrib.Connectors\Properties\VersionInfo.cs"
 			AssemblyVersion="$(BUILD_RELEASE)" AssemblyInformationalVersion="$(DECIMAL_BUILD_NUMBER)" />
+        <AssemblyInfo CodeLanguage="CS"
+			OutputFile="..\src\Umbraco.Deploy.Contrib.Export\Properties\VersionInfo.cs"
+			AssemblyVersion="$(BUILD_RELEASE)" AssemblyInformationalVersion="$(DECIMAL_BUILD_NUMBER)" />
```

Comment:
Is this going to tie the versions of `UmbracoDeploy.Contrib` and `UmbracoDeploy.Export` together?  It could be we need to release more versions of the latter if anyone runs into issues, but it's unlikely we want to update the former.  So would be good if they could be released independently.

If that presents difficulties with the current build process, maybe it makes sense to create a new, private repo and put the code in that?

Reply:
It would yes, but since the build pipeline doesn't work anymore, it would make sense to decouple the projects again and do a manual build...